XLS-0065d-single-asset-vault/README.md

```diff
@@ -0,0 +1,651 @@
+| `VaultID`         | :heavy_check_mark: |       `string`       |   `HASH256`   |     `N/A`     | The ID of the vault to which the assets are deposited. |
+| `Amount`          | :heavy_check_mark: | `string` or `object` |   `AMOUNT`    |     `N/A`     | Asset amount to deposit.                               |
+
+##### 3.2.1.1 Failure conditions
+
+- `Vault` object with the `VaultID` does not exist on the ledger.
+- The asset type of the vault does not match the asset type the depositor is depositing.
+- The depositor does not have sufficient funds to make a deposit.
+- Adding the `Amount` to the `AssetsTotal` of the vault would exceed the `AssetsMaximum`.
+- The `Vault` `lsfPrivate` flag is set and the `Account` depositing the assets does not have credentials in the permissioned domain.
+
+- The `Vault.Asset` is `MPT`:
+
+  - `MPTokenIssuance.lsfMPTCanTransfer` is not set (the asset is not transferable).
+  - `MPTokenIssuance.lsfMPTLocked` flag is set (the asset is globally locked).
+  - `MPToken(MPTokenIssuanceID, AccountID).lsfMPTLocked` flag is set (the asset is locked for the depositor).
+  - `MPToken(MPTokenIssuanceID, AccountID).MPTAmount` < `Amount` (insufficient balance).
+
+- The `Asset` is an `IOU`:
+  - The `lsfGlobalFreeze` flag is set on the issuing account (the asset is frozen).
+  - The `lsfHighFreeze` or `lsfLowFreeze` flag is set on the `RippleState` object between the Asset `Issuer` and the depositor.
+  - The `RippleState` object `Balance` < `Amount` (insufficient balance).
+
+##### 3.2.1.2 State Changes
+
+If no `MPToken` object exists for the depositor, create one. For object details, see [3.4.2 `MPToken`](#3.4.2-MPToken).
+
+- Increase the `MPTAmount` field of the share `MPToken` object of the `Account` by $\Delta_{share}$.
+- Increase the `OutstandingAmount` field of the share `MPTokenIssuance` object by $\Delta_{share}$.
+- Increase the `SharesTotal` of the `Vault` object by $\Delta_{share}$.
+
+- Increase the `SharesTotal` by $\Delta_{share}$.
+- Increase the `AssetsTotal` and `AssetsAvailable` by `Amount`.
+
+- If the `Vault.Asset` is `XRP`:
+
+  - Increase the `Balance` field of _pseudo-account_ `AccountRoot` by `Amount`.
+  - Decrease the `Balance` field of the depositor `AccountRoot` by `Amount`.
+
+- If the `Vault.Asset` is an `IOU`:
+
+  - Increase the `RippleState` balance between the _pseudo-account_ `AccountRoot` and the `Issuer` `AccountRoot` by `Amount`.
+  - Decrease the `RippleState` balance between the depositor `AccountRoot` and the `Issuer` `AccountRoot` by `Amount`.
+
+- If the `Vault.Asset` is an `MPT`:
+
+  - Increase the `MPToken.MPTAmount` by `Amount` of the _pseudo-account_ `MPToken` object for the `Vault.Asset`.
+  - Decrease the `MPToken.MPTAmount` by `Amount` of the depositor `MPToken` object for the `Vault.Asset`.
+
+- Increase the `AssetsTotal` and `AssetsAvailable` of the `Vault` by $\Delta_{asset}$.
```

Comment:
Is this the same as line 479? I think this phrasing is better.

Reply:
I'm not sure what you're referring to.

Reply:
Is (now) line 478 the same as (now) line 495? Manually copied below:

```
478 - Increase the `AssetsTotal` and `AssetsAvailable` by `Amount`.
495 - Increase the `AssetsTotal` and `AssetsAvailable` of the `Vault` by $\Delta_{asset}$.
```

Reply:
Thanks John. I updated to: 'Increase the `AssetsTotal` and `AssetsAvailable` of the `Vault`  by `Amount`'.